src/core/stdc/math.d

```diff
@@ -3378,25 +3378,25 @@ else version(DragonFlyBSD)
     double cbrt(double x);
```

Comment:
`cbrt` not pure?

Reply:
Added pure to the cbrt function... Thanks for checking !